eremetic_test.go

```diff
@@ -24,6 +25,24 @@ func TestMain(t *testing.T) {
 		})
 	})
 
+	Convey("GetSchedulerSettings", t, func() {
+		Convey("Contains defaults", func() {
+			u, err := user.Current()
+			So(err, ShouldBeNil)
+			s := getSchedulerSettings()
+			So(s.MaxQueueSize, ShouldEqual, 100)
+			So(s.Master, ShouldEqual, "")
+			So(s.FrameworkID, ShouldEqual, "")
+			So(s.CredentialFile, ShouldEqual, "")
+			So(s.Name, ShouldEqual, "Eremetic")
+			So(s.User, ShouldEqual, u.Username)
```

Comment:
Interesting discovery this defaults to "keis" on my system. Makes sense because we enable variables to be set from environment and USER is always set :)